### PR TITLE
feat(TDP-4277) : add min and sec in compute time since

### PR DESCRIPTION
**Pull request by tlnd-cgall**
**Link to the JIRA issue**
https://jira.talendforge.org/browse/TDP-4277

**Please check if the PR fulfills these requirements**
- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] The code coverage on new code is > 75 % for backend and > 95% for frontend
- [ ] The new code does not introduce new technical issues (sonar / eslint)
- [ ] Functional tests have been performed
- [ ] Docker configuration files for config-std or config-cloud profiles are impacted

**Please check the browsers you've tested on**
- [ ] Chrome, Firefox, Safari, Edge, IE11
- [ ] No, that's bad, this PR should not be merged !
- [ ] No, and no need to (backend changes only)

### CODE REVIEW

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/43/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/43/)

File: dataprep-actions/src/main/resources/org/talend/dataprep/i18n/actions_messages.properties [hunk 1]
```diff
@@ -513,9 +515,9 @@ parameter.specific_date.desc=Set here the date to compare to
 choice.now_server_side=Now
 choice.specific_date=Specific date
 
-choice.YEARS=Years
-choice.MONTHS=Months
-choice.DAYS=Days
+choice.YEARS=Years
```
Comment (smallet) [inline dataprep-actions/src/main/resources/org/talend/dataprep/i18n/actions_messages.properties]: There are some duplicates
choice.years & choice.YEARS
same for months, days

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/46/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/46/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/47/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/47/)

**Pull request merged**